Cannot compile to wasm

```
> GOOS=js GOARCH=wasm go build -o main.wasm ./cmd/wasm

# github.com/arnodel/golua/lib/oslib
../../../go/pkg/mod/github.com/arnodel/golua@v0.1.0/lib/oslib/clock.go:14:14: undefined: syscall.Getrusage
../../../go/pkg/mod/github.com/arnodel/golua@v0.1.0/lib/oslib/clock.go:14:32: undefined: syscall.RUSAGE_SELF
```
It's just simple hello world.

--- a/lib/oslib/clock.go
+++ b/lib/oslib/clock.go
@@ -1,5 +1,5 @@
-//go:build !windows
-// +build !windows
+//go:build !windows && !js
+// +build !windows,!js
 
 package oslib
 

--- a/lib/oslib/clock_fallback.go
+++ b/lib/oslib/clock_fallback.go
@@ -1,5 +1,5 @@
-//go:build windows
-// +build windows
+//go:build windows || js
+// +build windows js
 
 package oslib
 

